src/components/CvPagination/CvPagination.vue

```diff
@@ -2,12 +2,12 @@
   <div :class="`cv-pagination ${carbonPrefix}--pagination`" data-pagination>
```

Comment:
side note: pagination styling seem to fail
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/20342514/372b268d-4bd0-4f80-aa2c-ae85ea33585f)

Reply:
This is unrelated to this PR but I "fixed" it. 
@felipebritor looks like you added this to `.storybook/styles.scss` 
```css
  // set space between form items
  .bx--form-item {
    margin-bottom: 2rem;
  }
```
I'm sure I broke something by removing it so maybe have a look and add it to a story instead of to the style?